__tests__/app.test.js

```diff
@@ -199,6 +199,41 @@ describe("GET /api/articles/:article_id (plus comment_count)", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of the article_id's comments", async () => {
+    const article_id = 1;
+    const res = await request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200);
+    res.body.comments.forEach((comment) => {
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+        author: expect.any(String),
+        body: expect.any(String),
+      });
+    });
+  });
+  test("404: shows correct error status code and message when given an invalid article_id", async () => {
```

Comment:
This test is technically sound, but consider the wording of your description. 9999 isn't necessarily an invalid article_id - it might exist at some point in the future if enough comments are added to the db. It's just that it doesn't exist right now.

Reply:
Yeah yeah true, I will change wording now. Something on the lines of "when given correct a valid invalid_id but does not exist in database"

Reply:
This one still looks the same to me! The Id IS valid, as in, the type it is expecting IS a number, it just doesn't exist. So we could go for something as simple as "when given valid id type that does not exist in database"